nix/ext/pgvector.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, postgresql }:
+
+stdenv.mkDerivation rec {
+  pname = "pgvector";
+  version = "0.5.1";
```

Comment:
bump to 0.6.0

Reply:
resolved with https://github.com/supabase/postgres/pull/909/commits/28fa350acf5f791e7efedf7c45ed31c428872807